includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -338,19 +338,15 @@ public function pull( $items ) {
 	 *
 	 * This let's us grab all the IDs of posts we've PULLED from a given site
 	 *
-	 * @param array $item_id_mappings Mapping to log; key = origin post ID, value = new post ID.
-	 * @param int   $blog_id Blog ID
+	 * @param array   $item_id_mappings Mapping to log; key = origin post ID, value = new post ID.
+	 * @param int     $blog_id Blog ID
+	 * @param boolean $overwrite Whether to overwrite the sync log for this site. Default false.
 	 * @since 0.8
 	 */
-	public function log_sync( array $item_id_mappings, $blog_id = 0 ) {
+	public function log_sync( array $item_id_mappings, $blog_id = 0, $overwrite = false ) {
```

Comment:
@jeffpaul I think the logic for `$overwrite` is missing from this function. Currently the post ID is set to false which means it still shows on the skipped screen.

If an empty array is passed to `$item_id_mappings` it will fallback to `$current_site_log` instead, but it should overwrite.

Reply:
@elliott-stocks Ah, nice catch. I had refactored this a bit and in the process I ended up breaking things. I believe it should be working properly now

Reply:
Looks good!